sections/Chapter 5 Sprint 3/db_design_redone.tex

```diff
@@ -0,0 +1,148 @@
+\section{Database redesign}
+
+As previously mentioned in section \ref{currentState}, the code from last year was left in an unfinished and undocumented state. We estimated that it would be faster and easier to scrap it and start over rather than spending time trying to understand and document it.
```

Comment:
```suggestion
As previously mentioned in section \ref{currentState}, the code from last year was left in an unfinished and undocumented state. We estimated that starting over would give more value to the pipeline, rather than spending time trying to understand and document it.
```